apps/core/messages/en.json

```diff
@@ -190,7 +190,9 @@
       "alternativeAddress": "Alternative",
       "addNewAddress": "Add new address",
       "editButton": "Edit",
-      "deleteButton": "Delete"
+      "deleteButton": "Delete",
+      "confirmDeleteAddress": "Delete address",
+      "deleteModalTitle": "Are you sure you want to delete Address"
```

Comment:
🍹 We can probably change this to `Are you sure you want to delete address?` or `Are you sure?` if we show the `Delete Address` somewhere within the dialog

Reply:
Thanks, Anudeep. It's updated.